### DISCUSSION TITLE
[Project homepage] - Quick link name can be duplicated

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#62] [Q&A] [answered]
Hi guys, I noticed that the quick link name can be duplicated.
Is this the wanted behavior?

![Schermata 2022-10-21 alle 12 15 55](https://user-images.githubusercontent.com/28152586/197173250-a7cf14fe-854a-4d8d-a558-672b5f2a9365.png)

**Comment by fredmaggiowski** [comment, marked as answer]
Yes it is, they are names, not identifiers, as such you can have as many docs as you want :)

**Reply by frengooo** [reply]
Thanks neo-Mexican guy!
